docs/tutorials/UKFast_SafeDNS.md

```diff
@@ -0,0 +1,208 @@
+# Setting up ExternalDNS for Services on UKFast's SafeDNS
+
+This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster using SafeDNS
+
+Make sure to use **>=FIXME** version of ExternalDNS for this tutorial.
```

Comment:
@rjhenry can ypu replace the fixme with `0.11.0`? It will be unreleased when we merge this, but it is the next tag for release. Once you change that we're good to merge this one.